src/applications/personalization/dashboard/sass/dashboard-alert.scss

```diff
@@ -0,0 +1,122 @@
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
+@import "~@department-of-veterans-affairs/formation/sass/modules/m-nav-linkslist";
+@import "../../../../platform/forms/sass/_m-applications";
+
+$symbol-exclamation-circle: "\f06a";
+$symbol-info-circle: "\f05a";
+$symbol-checkmark-circle: "\f058";
+$symbol-pause-circle: "\f28b";
+$symbol-circle: "\f111";
```

Comment:
We can directly reference the [SASS variables](https://github.com/department-of-veterans-affairs/vets-website/blob/e69ce1d3c4238b070933690e99e2e008dc962f24/src/platform/site-wide/sass/iconography.scss#L159) for the FA icons to save a few lines.